README.md

```diff
@@ -2,12 +2,16 @@
 
 ## Introduction
 
+### ODK Collect
+
 ODK Collect is an open source Android app that replaces paper forms used in survey-based data
 gathering. It supports a wide range of question and answer types, and is designed to work well
 without network connectivity.
 
 You can read more about ODK Collect [here](https://docs.getodk.org/collect-intro/).
 
+### ODK Collect Extension
+
```

Comment:
We need to list and document the additional settings we have created apart from those provided by the collect app, for example, theme, form flow modifications, etc.

Reply:
Custom theming on form screens, Callbacks on form submission etc have been mentioned. Detailed documentation would be in the wiki